enhancements/bare-metal-fulfillment/README.md

```diff
@@ -0,0 +1,289 @@
+---
+title: bare-metal-fulfillment
+authors:
+  - Tzu-Mainn Chen
+creation-date: 2025-09-08
+last-updated: 2025-09-15
+tracking-link:
+  - None
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Bare Metal Fulfillment
+
+## Summary
+
+Bare metal fulfillment refers to the process through which a tenant acquires bare metal hosts and configures their networking.
+Afterwards, the tenant will be able to perform selected operations on the host: inventory information, power control, and
+serial console access. Any further operations - for example, image-based provisioning - are outside the scope of the O-SAC
+solution and must be integrated by the cloud provider.
+
+We believe this feature set is generic enough to support a wide range of desired bare metal cluster use cases, including:
+
+* support for an environment where developers have to flexibility to install their own operating systems and software
+* support for alternative clusters such as SLURM
+* support for bare metal configuration during O-SAC cluster fulfillment
+
+In this context, a “tenant” is a person with admin rights to the requested bare metal resources; at the MOC, this person would be
+a ColdFront PI or manager.
+
+## Motivation
+
+Bare metal fulfillment is a fundamental requirement for O-SAC, identified as an explicit need for the MOC. It will also serve
+as the basis of bare metal configuration for other fulfillment workflows, such as OpenShift cluster fulfillment.
+
+### User Stories
+
+* As a provider, I want to be able to define the available bare metal resource classes.
+* As a provider, I want to mark hosts as available for bare metal fulfillment.
+* As a tenant, I want to be able to see available bare metal resource classes.
+* As a tenant, I want to be able to select hosts for a bare metal cluster by resource class.
+* As a tenant, I want to be able to view inventory details for my hosts.
+* As a tenant, I want to be able to connect to the serial console of my hosts.
+* As a tenant, I want to be able to perform basic power control (power on/power off/reset) of my hosts.
+* As a tenant, I want to be able to attach hosts to existing networks.
```

Comment:
Maybe too implementation specific, but the user also probably wants to customize the L1 part of the connection, specifically whether to aggregate ports and apply the vlans on the port channel, or on the individual ports.